### PR TITLE
Bug: Make the result responsive

### LINKED ISSUE #7: Improve mobile usability
The card that displays the weather information isn't responsive. It's far too narrow on mobile, and too wide on desktop. Can you look into making it be an appropriate size on all screen sizes.

### PR DESCRIPTION
**Pull request by rk119**
Fixes #7 

Added media queries for making the result area responsive for different screen resolutions.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -21,12 +21,24 @@ input {
 
 .Results {
   background-color: white;
-  margin-left: 150px;
-  margin-right: 150px;
+  margin: auto;
+  width: 60%;
+  /* margin-right: 150px; */
```
Comment (irsayvid) [inline src/App.css]: Remove the commented line as it's not best practice

**Comment by irsayvid** [comment]
@rk119 @Neelaksh-Singh the changes look good. Please remove the commented code which is not needed and we'll be good to go.

**Review by irsayvid — APPROVED** [review]

**Comment by irsayvid** [comment]
LGTM!

**Comment by akshitadixit** [comment]
Please fix conflicts!

**Review by irsayvid — APPROVED** [review]

**Pull request merged**